src/command/edit.cpp

```diff
@@ -697,8 +697,8 @@ static void duplicate_lines(agi::Context *c, int shift) {
 					new_diag->Start = c->videoController->TimeAtFrame(cur_frame, agi::vfr::START);
 				}
 				else {
-					old_diag->Start = c->videoController->TimeAtFrame(cur_frame + 1, agi::vfr::START);
-					new_diag->End = c->videoController->TimeAtFrame(cur_frame, agi::vfr::END);
+					old_diag->End = c->videoController->TimeAtFrame(cur_frame, agi::vfr::END);
+					new_diag->Start = c->videoController->TimeAtFrame(cur_frame + 1, agi::vfr::START);
```

Comment:
I think the seemingly odd behavior here was actually deliberate because it made some case work better, but I need to double-check.

Reply:
If there is a reason I'd be curious, since I can't think of anything and it's definitely odd behavior.
https://github.com/Aegisub/Aegisub/commit/0ef9963782c637d6a8afd1282a8d49deb86c07b8#r7070919
tophf brought it up here too, though, and you never changed it, so I'm not sure what's up.